django/core/management/commands/migrate.py

```diff
@@ -235,6 +235,15 @@ def handle(self, *args, **options):
                         "migrations, and then re-run 'manage.py migrate' to "
                         "apply them."
                     ))
+                    for app_label, app_migrations in changes.items():
+                        self.stdout.write(self.style.WARNING(
+                            " Migrations for \'{}\':".format(app_label)
+                        ))
+                        for migration in app_migrations:
+                            for operation in migration.operations:
+                                self.stdout.write(self.style.WARNING(
+                                    "  - {}".format(operation.describe())
+                                ))
```

Comment:
IMO we don't need a list of all operations in this message, it's too detailed. If someone would like to check the list of operations they can always use `makemigrations --dry-run`.

Adding a list of apps should be enough, e.g.
```diff
diff --git a/django/core/management/commands/migrate.py b/django/core/management/commands/migrate.py
index 81696f11c1..32b1462319 100644
--- a/django/core/management/commands/migrate.py
+++ b/django/core/management/commands/migrate.py
@@ -227,8 +227,9 @@ class Command(BaseCommand):
                 changes = autodetector.changes(graph=executor.loader.graph)
                 if changes:
                     self.stdout.write(self.style.NOTICE(
-                        "  Your models have changes that are not yet reflected "
-                        "in a migration, and so won't be applied."
+                        "  Your models in apps [%s] have changes that are not "
+                        "yet reflected in a migration, and so won't be "
+                        "applied." % ', '.join(changes)
                     ))
                     self.stdout.write(self.style.NOTICE(
                         "  Run 'manage.py makemigrations' to make new "
```

Reply:
Thank you for the review @felixxm I'll make the adjustments right away :)

Reply:
Fixed the code to only list app names. Also @felixxm could you please elaborate on the kind of tests necessary for this?

Reply:
We should be able to assert `stdout`. You can find similar tests in `tests/migrations/test_commands.py`.

Reply:
I've added the tests please review it :)